### ISSUE TITLE
Asset compilation fails when updating to 6.5.3 (Mini CSS Extract Plugin initialization error)

### ISSUE THREAD

**Issue by tagliala** [shakacode/shakapacker#198]
Hi, I think it is this change:

https://github.com/shakacode/shakapacker/blob/0750276b2d686252cb3fb1dce8a3e67ba3a782a3/package/environments/base.js#L76

### CI Failure

https://github.com/diowa/ruby3-rails7-bootstrap-heroku/pull/270

### Error

```
$ ./bin/webpacker
[webpack-cli] Failed to load '~/dev/ruby3-rails7-bootstrap-heroku/config/webpack/webpack.config.js' config
[webpack-cli] Invalid options object. Mini CSS Extract Plugin has been initialized using an options object that does not match the API schema.
 - options.chunkFilename should be one of these:
   non-empty string | function
   -> This option determines the name of non-entry chunk files.
   -> Read more at https://github.com/webpack-contrib/mini-css-extract-plugin#chunkfilename
   Details:
    * options.chunkFilename should be a non-empty string.
    * options.chunkFilename should be an instance of function.
```

### How to replicate

```bash
$ rails _7.0.4_ new minicss-shakapacker --skip-active-record --skip-javascript
$ cd minicss-shakapacker/
$ echo "gem 'shakapacker', '6.5.3'" >> Gemfile
$ bundle exec rails webpacker:install
$ yarn add css-loader style-loader mini-css-extract-plugin css-minimizer-webpack-plugin
$ bin/webpacker
```

**Comment by Eric-Guo** [comment]
My #202 try to fix it, hopefully.

**Comment by tagliala** [comment]
Closed via #202 and released in 6.5.4
